src/plugins/content/accessibility/keyNavigation/helpers.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016-2020 (original work) Open Assessment Technologies SA ;
+ */
+import $ from 'jquery';
+import _ from "lodash";
+import strategyFactory from 'taoQtiTest/runner/plugins/content/accessibility/keyNavigation/strategiesManager';
+
+/**
+ * When either an element or its parents have this class - navigation from it would be disabled.
+ *
+ * @type {String}
+ */
+const ignoredClass = 'no-key-navigation';
+
+/**
+ * Checks whether element is navigable from
+ *
+ * @param {HTMLElement|keyNavigator} element
+ * @returns {boolean}
+ */
+export function allowedToNavigateFrom(element) {
+    if (element.getCursor) {
+        const {navigable} = element.getCursor();
+        element = navigable;
```

Comment:
You're right, this might introduce an issue in complex functions.
This parameter reassignment was introduced to fix an issue within the helper, and I chose to not use a local value to simplify the code. Anyway, I'll update it to prevent further issues if someone add more complexity in it later.